ui/imports/shared/views/AssetsView.qml

```diff
@@ -71,7 +71,16 @@ Item {
             readonly property string balance: !!modelData ? "%1".arg(modelData.enabledNetworkBalance.amount) : "" // Needed for the tests
             errorTooltipText_1: !!modelData && !! networkConnectionStore ? networkConnectionStore.getBlockchainNetworkDownTextForToken(modelData.balances) : ""
             errorTooltipText_2: !!networkConnectionStore ? networkConnectionStore.getMarketNetworkDownText() : ""
-            subTitle: !modelData || (!!networkConnectionStore && !networkConnectionStore.noTokenBalanceAvailable) ? "" :  LocaleUtils.currencyAmountToLocaleString(modelData.enabledNetworkBalance)
+            subTitle: {
```

Comment:
subTitle: {
                if (!modelData || ((networkConnectionStore && networkConnectionStore.noTokenBalanceAvailable))) {
                    return ""
                }
                return LocaleUtils.currencyAmountToLocaleString(modelData.enabledNetworkBalance)
            }

Reply:
If you don't mind i ll keep the more verbose option as optimising code size was probably the root cause of problem here